doc/modules/cross_validation.rst

```diff
@@ -209,6 +209,19 @@ The following sections list utilities to generate indices
 that can be used to generate dataset splits according to different cross
 validation strategies.
 
+Cross-validation iterators for i.i.d. data.
+===========================================
+
+Data is Independent Identically Distributed (i.i.d.) if the individual samples
+of the data are independent of each other and are drawn from an identical
+distribution.
+
+Simply put, it means that each data point is created equally, and there is no
+covariance between the different data points.
+
+The following cross-validators can be used in such case to get an unbiased
+performance estimation.
```

Comment:
There is a bias in k-fold cross-validation: it is slightly pessimistic and the lower the value of k the higher the bias (assuming the iid hypothesis holds of course).

I would just remove that sentence.